core/src/replay_stage.rs

```diff
@@ -2308,6 +2316,12 @@ impl ReplayStage {
                         warn!("cost_update_sender failed sending bank stats: {:?}", err)
                     });
 
+                // finalize block's min prioritization fee cache for this bank
+                block_min_prioritization_fee_cache
+                    .write()
+                    .unwrap()
+                    .finalize_block(bank.slot());
```

Comment:
would be good to move this into a separate thread because it does do an expensive filter, maybe the `cost_update_sender`?

Reply:
Good idea, I'll make change